<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>snifferClass</class>
 <widget class="QMainWindow" name="snifferClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>680</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1013</width>
    <height>680</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1013</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>sniffer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/LAI/Desktop/FO4N5KGVQAErHRH.jpg</normaloff>C:/Users/LAI/Desktop/FO4N5KGVQAErHRH.jpg</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="showPacketList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>521</width>
      <height>461</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="columnCount">
     <number>0</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QTextBrowser" name="showData">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>110</y>
      <width>431</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>clear capture window</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filterInput">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>40</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label01">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>filter expression:</string>
    </property>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>status:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="selectAdapter">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>select adapter</string>
    </property>
   </widget>
   <widget class="QLabel" name="adapterName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>961</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="CpatureButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>capture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submitFilter">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>70</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>submit filter</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1013</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sniffer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>showPacketList</sender>
   <signal>cellClicked(int,int)</signal>
   <receiver>snifferClass</receiver>
   <slot>showInformation(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>472</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>snifferClass</receiver>
   <slot>clearAllInformation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>637</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>submitFilter</sender>
   <signal>clicked()</signal>
   <receiver>snifferClass</receiver>
   <slot>getFilterTxt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>575</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>634</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectAdapter</sender>
   <signal>clicked()</signal>
   <receiver>snifferClass</receiver>
   <slot>selectAdapter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>28</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CpatureButton</sender>
   <signal>clicked()</signal>
   <receiver>snifferClass</receiver>
   <slot>captureStart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeTxt()</slot>
  <slot>showTest()</slot>
  <slot>showInformation(int,int)</slot>
  <slot>clearAllInformation()</slot>
  <slot>getFilterTxt()</slot>
  <slot>testThreadButtonClicked()</slot>
  <slot>selectAdapter()</slot>
  <slot>captureStart()</slot>
 </slots>
</ui>
